MvvmCross-Plugins/PictureChooser/MvvmCross.Plugins.PictureChooser.Droid/MvxPictureChooserTask.cs

```diff
@@ -251,27 +251,20 @@ private Bitmap ExifRotateBitmap(ContentResolver contentResolver, Uri uri, Bitmap
             if (bitmap == null)
                 return null;
 
-            var exif = new ExifInterface(GetRealPathFromUri(contentResolver, uri));
-            var rotation = exif.GetAttributeInt(ExifInterface.TagOrientation, (int)Orientation.Normal);
-            var rotationInDegrees = ExifToDegrees(rotation);
-            if (rotationInDegrees == 0)
-                return bitmap;
-
-            using (var matrix = new Matrix())
+            using (var inputStream = contentResolver.OpenInputStream(uri))
             {
-                matrix.PreRotate(rotationInDegrees);
-                return Bitmap.CreateBitmap(bitmap, 0, 0, bitmap.Width, bitmap.Height, matrix, true);
-            }
-        }
+                var ei = new Android.Support.Media.ExifInterface(inputStream);
```

Comment:
Is there another class called `ExifInterface` since you are making an alias? I think it should be enough to just write `using Android.Support.Media;`